manager/controlapi/service.go

```diff
@@ -433,7 +433,7 @@ func (s *Server) validateNetworks(networks []*api.NetworkAttachmentConfig) error
 		if network == nil {
 			continue
 		}
-		if network.Spec.Internal {
+		if network.Spec.Ingress {
```

Comment:
True, for ingress network created with older version, this check is not effective. Will fix thanks.